graphstorm-processing/graphstorm_processing/graph_loaders/row_count_utils.py

```diff
@@ -405,6 +420,14 @@ def verify_metadata_match(graph_meta: Dict[str, Dict]) -> bool:
         True if all row counts match for each type, False otherwise.
     """
     logging.info("Verifying features and structure row counts match...")
+
+    # Check if required keys exist
+    required_keys = ["edge_data", "edges", "node_data"]
```

Comment:
Why there are both edges and edge_data here?

Reply:
Edges refers to the edges that describe the graph structure (src,dst) while edge_data are features, labels etc. for each edge type.

This is according to the DGL chunked data specification: https://docs.dgl.ai/guide/distributed-preprocessing.html#chunked-graph-format

The linked spec is not 100% up to date however and should be fixed.